pom.xml

```diff
@@ -2593,13 +2593,28 @@
         </exclusions>
       </dependency>
 
+      <dependency>
+        <groupId>org.apache.orc</groupId>
+        <artifactId>orc-format</artifactId>
+        <version>1.0.0</version>
+        <classifier>${orc.classifier}</classifier>
+        <scope>${orc.deps.scope}</scope>
+      </dependency>
```

Comment:
Hmm, why do we need to exclude `orc-format` from `orc-core` and add it as a separate dependency?

Reply:
Thank you for review. It's because Apache Spark needs to control `orc.classifier` here. Without this, the default dependencies are pulled in.